OpenEdXMobile/src/main/java/org/edx/mobile/util/CalendarUtils.kt

```diff
@@ -379,4 +380,59 @@ object CalendarUtils {
                 .setData(builder.build())
         fragment.startActivity(intent)
     }
+
+    /**
+     * Helper method used to check that the calendar if outdated for the course or not
+     *
+     * @param context - current [Context] of the application
+     * @param accountName Name of the calendar owner
+     * @param calendarTitle Title for the course calender
+     * @param courseDateBlocks Course dates events
+     *
+     * @return Calender Id if calender is outdated otherwise -1
+     *
+     */
+    @JvmStatic
+    fun isCalendarOutOfDate(
+        context: Context,
+        accountName: String,
+        calendarTitle: String,
+        courseDateBlocks: List<CourseDateBlock>
+    ): Long {
+        if (isCalendarExists(context, accountName, calendarTitle)) {
+            val calendarId = getCalendarId(context, accountName, calendarTitle)
+            if (compareEvents(context, calendarId, courseDateBlocks).not()) {
+                return calendarId
+            }
+        }
+        return -1
+    }
+
+    /**
+     * Method to get the current user account as the calender owner
+     *
+     * @param environment Relevant configuration environment.
+     *
+     * @return calender owner account
```

Comment:
* @return calendar owner account or "local_user"

Reply:
also, we can make `LOCAL_USER` a private const

Reply:
done.